EvidenceApi.Tests/V1/Factories/ResponseFactoryTest.cs

```diff
@@ -1,19 +1,59 @@
+using System;
+using System.Collections.Generic;
+using AutoFixture;
+using EvidenceApi.V1.Boundary.Response;
 using EvidenceApi.V1.Domain;
 using EvidenceApi.V1.Factories;
+using FluentAssertions;
 using NUnit.Framework;
 
 namespace EvidenceApi.Tests.V1.Factories
 {
     public class ResponseFactoryTest
     {
-        //TODO: add assertions for all the fields being mapped in `ResponseFactory.ToResponse()`. Also be sure to add test cases for
-        // any edge cases that might exist.
+        private readonly IFixture _fixture = new Fixture();
+
         [Test]
-        public void CanMapADatabaseEntityToADomainObject()
+        public void CanMapAnEvidenceRequestDomainObjectToAResponseObject()
         {
-            // var domain = new Entity();
-            // var response = domain.ToResponse();
-            //TODO: check here that all of the fields have been mapped correctly. i.e. response.fieldOne.Should().Be("one")
+            var documentType = new DocumentType() { Id = "passport", Title = "Passport" };
```

Comment:
Should we use type instead of var where we know what the type will be when declaring a new variable?